react-components/table-intro/index.json

```diff
@@ -15,35 +15,43 @@
       },
```

Comment:
Change "title": "Introducing Patternfly React Table", to "The Patternfly React table" to be consistent with all modules